src/util/redirects.ts

```diff
@@ -3,7 +3,7 @@ import MediaWiki from '../MediaWiki.js'
 import Downloader from '../Downloader.js'
 import { getArticlesByIds, getArticlesByNS } from './mw-api.js'
 
```

Comment:
I would delete that file and move the sole function `getArticleIds()` to `mw-api.ts`

Reply:
Done.